plugins/withViroIos.ts

```diff
@@ -16,9 +18,7 @@ const withViroPods = (config: ExpoConfig) => {
 
       fs.readFile(`${root}/Podfile`, "utf-8", (err, data) => {
         data = insertLinesHelper(
-          `  pod 'ViroReact', :path => '../node_modules/@viro-community/react-viro/ios/'
-    pod 'ViroKit_static_lib', :path => '../node_modules/@viro-community/react-viro/ios/dist/ViroRenderer/static_lib'
-            `,
+          `pod 'ViroKit', :path => '../node_modules/@viro-community/react-viro/ios/dist/ViroRenderer/'`,
```

Comment:
Replace previous `pod` configuration as per this comment 👉🏾 https://github.com/ViroCommunity/viro/issues/56#issuecomment-986437376